src/go/rpk/pkg/cli/cluster/partitions/list.go

```diff
@@ -0,0 +1,202 @@
+					completion = rr.BytesMoved * 100 / rr.PartitionSize
+				}
+				for _, r := range rr.NewReplicas {
+					newReplica = append(newReplica, r.NodeID)
+				}
+				for _, r := range rr.PreviousReplicas {
+					oldReplica = append(oldReplica, r.NodeID)
+				}
+				return struct {
+					NT             string
+					PartitionID    int
+					MovingFrom     []int
+					MovingTo       []int
+					Completion     int
+					PartitionSize  string
+					BytesMoved     string
+					BytesRemaining string
+				}{
+					nt,
+					rr.PartitionID,
+					oldReplica,
+					newReplica,
+					completion,
+					sizeFn(rr.PartitionSize),
+					sizeFn(rr.BytesMoved),
+					sizeFn(rr.BytesLeft),
+				}
+			}
+
+			types.Sort(response)
+
+			out.Header("Partition movements", true, true, func() {
+				headers := []string{"Namespace-Topic", "Partition", "Moving-from", "Moving-to", "Completion-%", "Partition-size", "Bytes-moved", "Bytes-remaining"}
+				tw := out.NewTable(headers...)
+				defer tw.Flush()
+				for _, tps := range response {
+					tw.PrintStructFields(f(&tps))
+				}
+			})
+
+			out.Header("Reconciliation statuses", all, true, func() {
+				var j int
+				for _, p := range response {
+					fmt.Printf("%s\n", p.Ns+"/"+p.Topic+"/"+strconv.Itoa(p.PartitionID))
+					headers := []string{"Node-id", "Core", "Type", "Retry-number", "Revision", "Status"}
+					tw := out.NewTable(headers...)
+					for _, rs := range p.ReconciliationStatuses {
+						var row []interface{}
+						row = append(row, rs.NodeID)
+						for _, s := range rs.Operations {
+							row = append(row, s.Core, s.Type, s.RetryNumber, s.Revision, s.Status)
+						}
+						tw.Print(row...)
+					}
+					tw.Flush()
+					j++
+					if j < len(response) {
+						fmt.Println()
+					}
+				}
+			})
+		},
+	}
+
+	cmd.Flags().BoolVarP(&all, "print-all", "a", false, "Print how much data moved and remaining in bytes")
+	cmd.Flags().BoolVarP(&human, "human-readable", "H", false, "Print the partition size in a human-readable form")
+	cmd.Flags().StringSliceVarP(&partitions, "partition", "p", nil, "Partitions to print the ongoing movements (repeatable)")
+
+	return cmd
+}
+
+// This function returns true when a partition that movement is
+// ongoing is a requested partition by the --partition option.
+func contains(pReq []string, pRes string) bool {
+	for _, p := range pReq {
+		if p == pRes {
+			return true
+		}
+	}
+	return false
+}
+
+const helpListMovement = `Show ongoing partition movements.
+
+By default this command lists all the ongoing partition movements in the cluster.
+
+    rpk cluster partitions move-status
+
+The output contains the following columns, where PARTITION-SIZE is in bytes.
+Using -H, it prints the partition size in a human-readable format
+
+    NAMESPACE-TOPIC
+    PARTITION
+    MOVING-FROM
+    MOVING-TO
+    COMPLETION-%
+    PARTITION-SIZE
+    BYTES-MOVED
+    BYTES-REMAINING
+
+This command optionally adopts topic(s) to list the movement status of the specific
```

Comment:
I'm not really understanding the first sentence here -- but I think it can just be entirely dropped.

Reply:
Oh, in that case, wdyt about moving this to the top. Proposal:

```
By default this command lists all the ongoing partition movements in the cluster.
Topics can be specified to print the move status of specific topics. By default,
this command assumes the "kafka" namespace, but you can use a "namespace/" to
specify internal namespaces.

    rpk cluster partitions move-status
    rpk cluster partitions move-status foo bar kafka_internal/tx

The "--partition / -p" flag can be used with topics to additional filter
requested partitions:

    rpk cluster partitions move-status foo bar --partition 0,1,2

The output contains the following columns, where PARTITION-SIZE is in bytes.
Using -H, it prints the partition size in a human-readable format

    NAMESPACE-TOPIC
    PARTITION
    MOVING-FROM
    MOVING-TO
    COMPLETION-%
    PARTITION-SIZE
    BYTES-MOVED
    BYTES-REMAINING

Using "--print-all / -a" the command additionally prints "RECONCILIATION STATUSES"
which reveals internal states on how the ongoing reconciliations work for debugging
purposes. That is, reported errors don't necessarily mean real problems.
```

Reply:
Indeed. Thank you.